airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/command/DestinationConfiguration.kt

```diff
@@ -64,7 +64,7 @@ abstract class DestinationConfiguration : Configuration {
     /** Memory queue settings */
     open val maxMessageQueueMemoryUsageRatio: Double = 0.2 // 0 => No limit, 1.0 => 100% of JVM heap
     open val estimatedRecordMemoryOverheadRatio: Double =
-        0.1 // 0 => No overhead, 1.0 => 100% overhead
+        1.1 // 1.0 => No overhead, 2.0 => 100% overhead
```

Comment:
Other option is to adjust the math in the class, but this felt straight forward enough.